### PR TITLE
fix(large partition): increase scylla-bench and change scylla-bench event severity

### PR DESCRIPTION
**Pull request by juliayakovlev**
Few changes:

1. increase scylla-bench command timeout to prevent failure when the cluster is overloaded
2. descrease ScyllaBenchEvent from CRITICAL to ERROR when it failed because of timeout
3. don't print ScyllaBenchEvent with type='finish' if the scylla-bench failed
4. change workload from "uniform" to "sequential" because of for uniform we can't define duration
5. increase timeout to 300s for last command with "-validate-data" - increase the chance don't fail on truncate timeout (every "write" load with "-validate-data" first perform truncate table)

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: sdcm/cluster.py [hunk 1]
```diff
@@ -3845,11 +3845,16 @@ def node_run_stress_bench(node, loader_idx, stress_cmd, node_list):
                     log_file=log_file_name)
             except Exception as exc:  # pylint: disable=broad-except
                 errors_str = format_stress_cmd_error(exc)
-                ScyllaBenchEvent(type='failure', node=str(node), stress_cmd=stress_cmd,
-                                 log_file_name=log_file_name, severity=Severity.CRITICAL,
-                                 errors=[errors_str])
+                if "truncate: seastar::rpc::timeout_error" in errors_str or "Operation timed out" in errors_str:
```
Comment (fruch) [inline sdcm/cluster.py:3848]: I'm not exactly sure about this, if there is a timeout, during a run, we do want to stop a scylla bench if it's the main stress.

I don't know yet exactly how, I think we should have a way to set the priority/severity of specific stress command, and avoid this logic here.

Comment (juliayakovlev) [reply sdcm/cluster.py:3848]: Yes, I did it after discussion with @roydahan . Timeout on truncate operation is the known issue and we don't want to stop and fail the test on it. 
"Operation timed out" - I am waiting the answer from Roy, but added meanwhile

Comment (roydahan) [reply sdcm/cluster.py:3848]: when do we get "Operation timed out"?

Comment (juliayakovlev) [reply sdcm/cluster.py:3848]: > when do we get "Operation timed out"?

When failed to receive the answer from all nodes by CL

Comment (roydahan) [reply sdcm/cluster.py:3848]: but it's not the terminal failure that fails the stress, right?

Comment (juliayakovlev) [reply sdcm/cluster.py:3848]: @roydahan 
It may fail load exactly like "truncate timeout". The question here if we want to fail the whole test

Comment (roydahan) [reply sdcm/cluster.py:3848]: So for now, as long as we aren't aware of a known issue with "Operation timeout" let's remove it.

Comment (juliayakovlev) [reply sdcm/cluster.py]: Done

**Comment by roydahan** [comment]
I'm ok to merge and backport.
(@bentsi )

**Pull request merged**